astropy/modeling/tests/test_blackbody.py

```diff
@@ -32,8 +32,8 @@ def test_evaluate(self, temperature):
         b = BlackBody1D(temperature=temperature,
                         bolometric_flux=bolometric_flux)
 
-        assert_quantity_allclose(b(1.4 * u.micron), 4756726.111003904 * u.Jy)
-        assert_quantity_allclose(b(214.13747 * u.THz), 4756726.111003904 * u.Jy)
+        assert_quantity_allclose(b(1.4 * u.micron), 4734464.498937388 * u.Jy)
```

Comment:
That makes sense, but what then surprises me is that in the documentation table `L_sun` has not changed.